README.md

```diff
@@ -88,6 +88,59 @@ docker image rm <image-id>
 
 There is no need to remove postgres and java or similar core images.
 
+## Azure Service Bus & Local Testing
```

Comment:
Can delete the bits about ActiveMQ since no longer applicable when using the Azure lib?

Reply:
done